packages/mjml-section/src/index.js

```diff
@@ -100,7 +100,7 @@ export default class MjSection extends BodyComponent {
       },
       div: {
         ...(fullWidth ? {} : background),
-        Margin: marginAlignment,
```

Comment:
If it gets merged, you should revert this change. Some clients removes some css attributes depending on the case.

Reply:
👌